envoy/network/listener.h

```diff
@@ -164,7 +164,7 @@ class ListenerConfig {
    *         A listener that doesn't listen on a port can only receive connections
    *         redirected from other listeners.
    */
-  virtual bool bindToPort() PURE;
+  virtual bool bindToPort() const PURE;
```

Comment:
Is it necessary or we can get benefit from it?

Reply:
This is used by `bool hasDuplicatedAddress(const ListenerImpl& other) const` https://github.com/envoyproxy/envoy/pull/22005/files#diff-e7a517ffa60d492c4979c569cc2857ed4bdc89e9d188ae3167a777cdf264e189R322, then I have to change this method to const, since they just return the value of member, so I think it makes sense to mark it to const.

Reply:
Sound reasonable to me.